.github/workflows/docker.yml

```diff
@@ -1,7 +1,7 @@
 name: Docker
 on:
   push:
-    tags: ["*"]
+    # tags: ["*"]
     paths:
       - ".github/workflows/docker.yml"
       - "Dockerfile"
```

Comment:
Ha, this is clever! I may want to use something similar to simplify debugging & troubleshooting the release workflow for Zinnia - https://github.com/filecoin-station/zinnia/pull/91

Reply:
I stole it from l1-node 👍